src/DynamoApplications/StartupUtils.cs

```diff
@@ -236,6 +271,8 @@ public static DynamoModel MakeCLIModel(CommandLineArguments cmdLineArgs)
                 userDataFolder: cmdLineArgs.UserDataFolder,
                 commonDataFolder: cmdLineArgs.CommonDataFolder,
                 serviceMode: cmdLineArgs.ServiceMode);
+
+            ImportAssemblies(model, cmdLineArgs.ImportedPaths);
```

Comment:
Is there an extra import step happening here that wasn't happening before?

Reply:
This definitely sticks out as a strange place for this code to live. I would just expect this method to create a dynamo model?

Reply:
It seems like you are trying to get around the fact that you're creating a CLI model in design automation but you're not really using a CLI entry point?

Reply:
Well, why should it matter if the model is used by DynamoCLI or another CLI (in my case the DesignAutomationCLI).
My indention is to have a simple, clean way to build a model for the purposes of being used in a headless way

Reply:
The `cmdLineArgs` input (for `MakeCLIModel`) allows specifying ImportedPaths, but it is not used. 
There is no public API to import libs (that I could find). I could call the import API outside of the MakeModel call/..with  no need for this PR

Reply:
Is this clean?

This function doesn't do what it says on the tin anymore.
Won't this also cause the dynamo CLIs to attempt to import these assemblies twice- I guess that's probably, ok - though it could be slow?

why can't `DesignAutomationCLI` use the same set of flags the other CLI's do and just import assemblies that way?

Reply:
sorry, I did not notice the double import call

Reply:
evidently not simple or clear enough
I can simply try to expose the Import function...or the existing StartupDynamo method, and revert everythign else

Reply:
Part of the confusion is that the method MakeCLIModel has arguments that take no effect when you make the function call. Maybe it should be make clear that the importPaths are not meant for that function ...